static/docs/get-started/example-versioning.md

```diff
@@ -295,8 +305,16 @@ our example, please notice that `train.py` produces binary files (e.g.
 When you have a script that takes some data as an input and produces other data
 outputs, a better way to capture them is to use `dvc run`:
 
+> Please go back to the master branch code and data if you tried the commands in
+> the [Switching between versions](#switching-between-versions) section with:
+>
+> ```dvc
+> $ git checkout master
+> $ dvc checkout
+> ```
+
 ```dvc
-$ dvc remove -p model.h5.dvc
+$ dvc remove -pf model.h5.dvc
```

Comment:
typo?

Reply:
Hmm no, that was part of my review of actually trying the example. without `-f` the sample would need to include more the confirmation prompt and somehow indicate to the user to answer it.

Reply:
👍